mmv1/third_party/terraform/tests/resource_workstations_workstation_test.go.erb

```diff
@@ -0,0 +1,152 @@
+<% autogen_exception -%>
+package google
+<% unless version == "ga" -%>
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccWorkstationsWorkstation_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProvidersOiCS,
```

Comment:
```suggestion
		Providers:    testAccProviders,
```

`testAccProvidersOiCS` is used for examples, but we can just use `testAccProviders` here. That'll require removing `provider = google-beta` through tests in this file. This'll make moving to GA in the future simpler, since we'll just need to remove the version guard in the file.

Reply:
This is very nice and good to know! Fixed.